includes/abstracts/llms-abstract-generator-posts.php

```diff
@@ -542,35 +528,79 @@ protected function handle_reusable_blocks( $post, $raw ) {
 		// Importing blocks is disabled.
 		if ( ! $this->is_reusable_block_importing_enabled() ) {
 			return null;
+		} elseif ( empty( $raw['_extras']['blocks'] ) ) {
```

Comment:
What's the reasoning behind this change?

Reply:
`else` after a return statement doesn't make sense.
It's not wrong, but doesn't make sense, right?

Reply:
`else` by itself doesn't make sense but an `elseif` I think does.

If the first condition isn't met, there won't be a return and then the second condition might be met.

Reply:
the only relevant part of an `elseif` after a return statement is the `if` which I keep.
`elseif` is equivalent to `else { if (something) { do(); } }` so if you had something like:
```php
if ( a ) {
    return;
} else {
    if ( b ) {
        do_something_else();
   }
}
```
I'm sure you turned it into:
```php
if ( a ) {
    return;
} 
if ( b ) {
    do_something_else();
}
```
so that's what I'm proposing here :D